fuseimg/data/ops/aug/geometry.py

```diff
@@ -49,6 +49,8 @@ def __call__(
         :param interpolation: used PIL interpolation method see TTF.affine for details.
         :return: the augmented image
         """
+        if key not in sample_dict:
```

Comment:
Let's wrap with OpIfExist which applies an op if some key exists. WDYT?
You will need to implement it :-P

Reply:
I'll revert for now.
Not using it anyway.